regression/cbmc-java/enum2/test.desc

```diff
@@ -0,0 +1,10 @@
+KNOWNBUG
+Main.class
+--lazy-methods --stop-on-fail
+^VERIFICATION SUCCESSFUL$
+^EXIT=0$
+^SIGNAL=0$
+--
+^warning: ignoring
+--
+ordinal() not loaded by --lazy-methods
```

Comment:
Should there be an issue for this, or maybe is there one already in some project?